service/block/event.go

```diff
@@ -58,8 +58,8 @@ func (s *Service) handleRawBlock(raw *Raw) error {
 			"block hash", raw.Hash().String())
 		return err
 	}
-	// create ExtendedBlock
-	extendedBlock := &ExtendedBlock{
+	// create Block
+	extendedBlock := &Block{
 		data: extendedBlockData,
```

Comment:
What happens to the other fields? do they get stored separately (later/in followup PRs)?

Reply:
Yes.